configurationdemo/resources/projects.xml

```diff
@@ -164,49 +164,34 @@ To launch this sample application on the Kaa C SDK:
 		Click <b>Ok</b> and download the .tar.gz file.
 	</li>
 	<li>
-		Unpack the downloaded archive and run the following command from the directory containing the unpacked source files.
-			<pre>
-				./build.sh deploy
-			</pre>
+		Unpack the downloaded archive.
+	</li>
+	<li>
+		Unpack the SDK archive.
+        <pre>
+            tar zxf libs/kaa/kaa* -C libs/kaa
+        </pre>
 	</li>
 	<li>
-		The program will ask you to enter the target platform for your build.
-		Press Enter to specify the posix platform (default) or enter the name of one of the <a href="http://kaaproject.github.io/kaa/docs/v0.10.0/Programming-guide/Using-Kaa-endpoint-SDKs/Supported-platforms/" target="_blank">supported platforms</a> and press Enter.
-		The C SDK and the demo application are now built and the demo application is launched.
+        Create <b>build<\b> directory, run CMake and make:
+			<pre>
+                mkdir build
+                cd build
+                cmake ..
+                make
+            </pre>
+        By default the project is generated for Posix target.
```

Comment:
I see the explanation is present below, but that needs to be explicitely visible to user at the point of 'target' selection reference.